<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoadStyleClass</class>
 <widget class="QMainWindow" name="RoadStyleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RoadStyle</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>962</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionHighways"/>
    <addaction name="actionAvenues"/>
    <addaction name="actionStreets"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionProperty"/>
    <addaction name="actionReferenceRoads"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAll_roads">
   <property name="text">
    <string>All roads</string>
   </property>
  </action>
  <action name="actionHighways">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Highways</string>
   </property>
  </action>
  <action name="actionAvenues">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Avenues</string>
   </property>
  </action>
  <action name="actionStreets">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Streets</string>
   </property>
  </action>
  <action name="actionExtract_and_Save">
   <property name="text">
    <string>Extract and Save</string>
   </property>
  </action>
  <action name="actionProperty">
   <property name="text">
    <string>Property</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionReferenceRoads">
   <property name="text">
    <string>Reference Roads</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RoadStyle.qrc"/>
 </resources>
 <connections/>
</ui>
